docs/content/stepbystep/IDE-setup.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Setting up the IDE'
+metaTitle: 'IDE Setup | Discordeno'
+metaDescription: 'Setup VSC for discordeno'
+---
+
+# Installing
+
+There are many editors for js/ts, but I recommend Visual Studio Code as its free and fast with a lot of extensions (Including a Deno one)
+
+Here are the others:
```

Comment:
Add the link to the websites of each editor/IDE.
Example: `- [Atom](https://atom.io)`